mgmtsystem_action/mgmtsystem_action.py

```diff
@@ -45,6 +45,18 @@ class mgmtsystem_action(models.Model):
     system_id = fields.Many2one('mgmtsystem.system', 'System')
     company_id = fields.Many2one('res.company', 'System', default=own_company)
 
+    stage_id = fields.Many2one(
+        'mgmtsystem.action.stage',
+        'Stage',
+        default=lambda self: self.get_default_stage()
```

Comment:
I use lambda so `get_default_stage` can be overriden. AFAIK setting default to a member function directly does not allow it to be overriden the way that a call to `self.` does.